doc/daal4py/_templates/layout.html

```diff
@@ -0,0 +1,17 @@
+{% extends "!layout.html" %}
+{% block extrahead %}
+    <script defer type="text/javascript" src="https://www.intel.com/content/dam/www/global/wap/performance-config.js" ></script>
```

Comment:
what are implications of this layout config? can you please also render doc?

Reply:
It lets you build the daal4py docs directly from the folder. You can see the output at: https://aepanchi.github.io/intelpython.github.io/daal4py/